test/unit/app/controllers/transactions/recipient-blacklist-checker-test.js

```diff
@@ -22,7 +22,7 @@ describe('Recipient Blacklist Checker', function () {
     it('does not fail on test networks', function () {
       let callCount = 0
       const networks = [ROPSTEN_CODE, RINKEBY_CODE, KOVAN_CODE, GOERLI_CODE]
-      for (const networkId in networks) {
```

Comment:
I guess it doesn't fail on network IDs that don't exist either? 😬

Reply:
No, but that seems fine, this is a very specific blacklist 🤷‍♂️